weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md

```diff
@@ -3,15 +3,152 @@ Copyright (c) 2021, Oracle and/or its affiliates.
+
+# Properties
+
+## Categories
+
+* Primary category: Compute
+
+   * Subcategory: Application Infrastructure
+
+* Secondary category: Web
+
+   * Subcategory: Web Apps
+   
+## Industries
+
+* Checked Apps 
+
+* Not checked Microsoft Clouds for Industry
+
+* App version: Various
+
+## Legal
+
+* Checked Use the Standarard Contract for Microsoft's commercial marketplace?
+
+## Custom amendments...
+
+No content.
+
 # What is this stuff?
 
-Useful Links to documentation, video, blog of WebLogic on Azure IaaS.
+Useful Links to documentation, video, blog of WebLogic on Azure.
+
+# Offer Listing
+
+## Name
+
+Oracle WebLogic Server on Azure Solution Overview
+
+## Search results summary
+
+Oracle WebLogic Server is a scalable, enterprise-ready Java application server.
+
+## Description (Observed max 2973 characters, not including formatting)
+
+<p>Azure Oracle WebLogic Server (WLS) offers enable you to embrace cloud computing by making it as easy as possible to migrate your Java applications to Azure. The offers increase productivity by automating most boilerplate Java and Azure resource provisioning and configuration tasks so that you can focus on application development. The offers cover deployment to both Azure Virtual Machines and the Azure Kubernetes Service (AKS).</p>
+
+<p>The offers are linked in the <i>Learn more</i> section at the bottom of this page.</p>
+
+<p>These offers are Bring-Your-Own-License. They assume you have already procured the appropriate licenses with Oracle and are properly licensed to run offers in Azure.</p>
+
+<p><i>If you want to work closely on your migration scenarios with the engineering team developing these offers, just hit the CONTACT ME button</i>. Program managers, architects and engineers will reach back out to you shortly and initiate collaboration!</p>
+
+<p><b>WLS on Virtual Machines</b><br/>
+The WLS on virtual machines offers automate provisioning virtual network, storage, and Linux resources, installing WLS, setting up security with a network security group, easing database connectivity, configuring load-balancing with App Gateway or Oracle HTTP Server, connecting to Azure Active Directory, enabling centralized logging via ELK as well as integrating distributed caching with Oracle Coherence.</p>
+
+<p>There are several offers that target use cases such as single node with admin server enabled and WLS cluster (including dynamic cluster). These offers supports a range of operating system, Java and WebLogic versions such as WLS 14 and JDK 11 on Oracle Linux 7.6 through base images. These base images are also available on Azure on their own. The base images are suitable for customers that require very highly customized Azure deployments. The current set of base images are available <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=oracle%20weblogic%20base%20image&amp;page=1">in the Azure marketplace</a>.</p>
```

Comment:
admin server -> Administration server
(including dynamic cluster) -> (including both configured and dynamic clusters)